packages/react-instantsearch-theme-algolia/styles/_Panel.scss

```diff
@@ -0,0 +1,10 @@
+.ais-Panel__empty {
+  display: none;
```

Comment:
Fine with me. We can not enforce anything of the panel stylesheet, but decline it depending on the widget. 

Also, I didn't add the Panel feature to the MultiRanges regarding the fact that the list is provided manually by the user. Do you have something in mind for this widget?